### ISSUE TITLE
keyboard accessibility?

### ISSUE THREAD

**Issue by tespin** [evnbr/bindery#88]
hi - really grateful for this project. thanks for your time and energy.

i wanted to ask if bindery was keyboard accessible? specifically if the page turning was attached to any keyboard events. i'd like my bindery project to be navigable by folx with disabilities. here is a related page from the world wide web consortium:

https://www.w3.org/WAI/perspective-videos/keyboard/

**Comment by evnbr** [comment]
Hi @tespin, glad you are finding it valuable!

The default grid mode can be scrolled with the arrow keys, and if you have anchor tags in the content you can of course `tab` through them like any normal web page. You can press Cmd/Ctrl-P to print rather than click the button. 

However, the flipbook mode, view switcher, and various print settings ui don't have clear keyboard events.
I can add that to the roadmap.

Regarding accessibility more broadly— can you say more about how you're distributing your project? The focus of bindery is to generate PDFs or physically printed-on-paper content, and so I haven't spent much time on the preview itself. I would also point out that splitting content into separate pages violates almost every web-markup accessibility guideline there is— if you are aiming for on-screen content accessible to people using screen readers, I would not recommend using bindery for that.

**Comment by tespin** [comment]
thanks for clarifying @evnbr ! originally i was using it just to make a pdf, but i enjoyed the flipbook mode as an online alternative. it also seemed like a safer way to interact with my book during covid, so i was hoping to add the bindery project to a personal website.

i don't need the project to remain in flipbook mode, but it would be great if i could switch between grid / flipbook modes with keys.

**Comment by evnbr** [comment]
Hi @tespin — as of 2.3.6, you should be able to use left/right arrows to flip through pages in flipbook mode. You can also now see focus indicators on controls, so you can use `Tab` to navigate and change various dropdowns.

I think that makes all of bindery's ui keyboard navigable. let me know if i've missed any scenarios, and feel free to open a new issue if so.
